FastSurferCNN/data_loader/dataset.py

```diff
@@ -101,11 +97,10 @@ class MultiScaleDataset(Dataset):
     """
     Class for loading aseg file with augmentations (transforms)
     """
-    def __init__(self, dataset_path, cfg, gn_noise=False, transforms=None):
```

Comment:
Please revert - during training (in MultiScaleDataset and MultiScaleDatasetVal) gaussian noise is wanted and needed! Only during inference the lines are irrelevant/unneeded!